### ISSUE TITLE
linux/arm64 docker image not available?

### ISSUE THREAD

**Issue by LGmatrix13** [tursodatabase/libsql#899]
I've been using the libsql docker image and it seems like arm64 isn't recognized as a valid platform in my Docker file, where I have:

```
FROM --platform=linux/arm64  ghcr.io/tursodatabase/libsql-server:latest
```

Docker responds with the error:

![image](https://github.com/tursodatabase/libsql/assets/19254314/23825980-dc2e-4850-b98f-083e128941d4)

've noticed `arm64` is specified as a platform in the libsql [github publish workflow](https://github.com/tursodatabase/libsql/blob/6b46606a084d2c1ae468fd79bc047f944cbe9a82/.github/workflows/publish-server.yml#L30). But, docker is telling it is not a valid platform the image supports... Is there any reason why this might be the case?

**Comment by penberg** [comment]
@QuiiBz is the arm64 Docker image working for you after the merge?

**Comment by QuiiBz** [comment]
I didn't try it because I found out about `turso dev` which simplifies my workflow.

Looks like it's tagged as `unknown/unknown` instead of `linux/arm64`:
<img width="1623" alt="Screenshot 2024-01-16 at 15 05 38" src="https://github.com/tursodatabase/libsql/assets/43268759/805cffe3-a0bc-46a1-a5ee-49d8bc8fbb38">

**Comment by LGmatrix13** [comment]
@penberg looks like specifying `unknown/unknown` as a `platform` works as a "valid platform", but it results in an invalid tar header once it starts building:

![image](https://github.com/tursodatabase/libsql/assets/19254314/b7aa7d03-4f61-4b04-9733-e5ab89354e96)

**Comment by SpencerDuball** [comment]
I have run into this as well, it's possible this is because you are using an apple M1/M2 family chip. I found this blog post that helped resolve the issues until there is an image for this platform: https://devcoops.com/no-matching-manifest-for-linux-arm64-v8/

TLDR;
```bash
docker run -p 8080:8080 --platform=linux/amd64 ghcr.io/tursodatabase/libsql-server:latest
```
